.travis.yml

```diff
@@ -9,11 +9,13 @@ env:
 before_install:
   - brew update 
   - brew install carthage
+
+install:
   - carthage bootstrap --use-submodules --no-build
+  - pod repo update --silent
```

Comment:
Is this command still needed? (since `pod lib lint` for the local spec is now used)